File: examples/nodejs/src/updateVersion.js
import { post } from "./lib/sock.js";

(async () => {
  const payload = {
    message: {
      binary_url: "http://example.com/nodex-agent-1.0.0.zip",
      path: "/tmp",
    },
  };

  const response = await post("/internal/version/update", payload);

  console.log("The response is as follows.\n");
  console.log(response);
})();

File: examples/python/src/update_version.py
import json
from platform_os import is_windows


if is_windows():
    from request import post
else:
    from sock import post


payload = {
    "message": {
        "binary_url": "http://localhost:3000/nodex.zip",
    }
}

json_response = post("/internal/version/update", payload)

print("The response is as follows.\n")
print(json_response)
